docs/Getting-Started/Create-bootable-media/bootable-usb-using-windows.md

```diff
@@ -1 +1,93 @@
+
+![Download Etcher](/images/create-bootable-usb-win10/download_etcher.png)
+
+2. Navigate to the downloaded executable and run it.
+
+![Run Etcher Executable](/images/create-bootable-usb-win10/run-etcher-exe.png)
+
+3. Click `I Agree` and wait for the installation to complete.
+
+![Install Etcher](/images/create-bootable-usb-win10/install-etcher.png)
+
+4. Wait for the installation process to complete.
+
+![Complete Installation](/images/create-bootable-usb-win10/complete-install.png)
+
+## Download the Pop!_OS.iso File
+
+![Linux Download ISO](/images/create-bootable-usb-linux/using-linux-download-iso.png)
+
+Download the appropriate Pop!_OS.iso file for your target system. See the [preparation guide](/Getting-Started/Create-bootable-media/create-bootable-usb.html#choose-a-pop_os-image) for information about each version.
+
+Take note of the hash value for your chosen ISO. Use this value to verify the integrity of the ISO file.
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
+
+## Verify the Hash Value
+
+1. Launch the Windows Command Prompt.
+
+![Launch CMD](/images/create-bootable-usb-win10/launch-cmd.png)
+
+2. Navigate to the folder where the Pop!_OS.iso file is located. The user's Downloads folder is used in the below example. Substitute your username for `USERNAME`.
+
+```bash
+cd C:\Users\USERNAME\Downloads
+```
+
+3. Use the `CertUtil` program to verify your ISO file. Make sure the name of the ISO matches exactly what is in your folder. The filename changes depending on the ISO version you chose.
+
+```bash 
+CertUtil -hashfile pop-os_21.10_amd64_intel_8.iso sha256
+```
+
+4. This command will output a long string of characters. Compare the output to the appropriate hash value on the Pop!\_OS download site.
+
+## Create the Bootable USB
+
+Flash your USB media with the Pop!_OS.iso image. Note that this process will completely erase all previous data on the USB storage device.
+
+1. Launch Etcher if it is not already running.
+
+2. Select `Flash from file`, then navigate to and select your Pop!\_OS.iso file.
+
+![Flash From File](/images/create-bootable-usb-win10/flash-from-file.png)
+
+3. Click `Select target` and select your USB drive.
+
+![Select Target](/images/create-bootable-usb-win10/select-target.png)
+
+4. Click `Flash!`.
+
+![Click Flash](/images/create-bootable-usb-win10/click-flash.png)
+
+5. Close Etcher once the installation completes.
+
+![Click Finish](/images/create-bootable-usb-win10/click-finish.png)
+
+## Next Steps
+
+Use your bootable Pop!_OS USB to demo, install, or recover a current Pop!_OS installation.
+
+### Install Pop!_OS
+
+Power off the target computer and insert the bootable USB. Power on the computer and enter the boot device menu selection for your BIOS or UEFI system. Follow the steps in the [Standard Installation](/Getting-Started/Installation/installation.md) section.
```

Comment:
It might be helpfull to have some links to examples of how to bring up the boot menu. Here are a few.

https://support.system76.com/articles/boot-menu/

https://www.disk-image.com/faq-bootmenu.htm

https://support.fixmestick.com/hc/en-us/articles/360001753213-List-of-Boot-Menu-Hot-Keys

Reply:
I agree they were just meant to be examples.

Reply:
I'll add the table from that article. My understanding is that this document should be a resource that can be copied and viewed offline, so I'd like to avoid outside linking as much as possible.